packages/samples/common-types/openapi/v5/networksecurityperimeter.json

```diff
@@ -0,0 +1,518 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Network security perimeter common type definitions",
+    "version": "v5",
+    "description": "Common types for network security perimeters based on a shared API specification. These common, versioned type definitions are intended for resource providers (RPs, except Network RP) to use, and reuse, for defining their own API versions that share a set of type definitions that is consistent across providers.",
+    "contact": {}
+  },
+  "host": "management.azure.com",
+  "paths": {},
+  "definitions": {
+    "AccessRule": {
+      "type": "object",
+      "description": "Access rule in a network security perimeter configuration profile",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the access rule"
+        },
+        "properties": {
+          "$ref": "#/definitions/AccessRuleProperties"
+        }
+      }
+    },
+    "AccessRuleDirection": {
+      "type": "string",
+      "description": "Direction of Access Rule",
+      "enum": [
+        "Inbound",
+        "Outbound"
+      ],
+      "x-ms-enum": {
+        "name": "AccessRuleDirection",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Inbound",
+            "value": "Inbound",
+            "description": "Applies to inbound network traffic to the secured resources."
+          },
+          {
+            "name": "Outbound",
+            "value": "Outbound",
+            "description": "Applies to outbound network traffic from the secured resources"
+          }
+        ]
+      }
+    },
+    "AccessRuleProperties": {
+      "type": "object",
+      "description": "Properties of Access Rule",
+      "properties": {
+        "direction": {
+          "$ref": "#/definitions/AccessRuleDirection"
+        },
+        "addressPrefixes": {
+          "type": "array",
+          "description": "Address prefixes in the CIDR format for inbound rules",
+          "items": {
+            "type": "string"
+          }
+        },
+        "subscriptions": {
+          "type": "array",
+          "description": "Subscriptions for inbound rules",
+          "items": {
+            "$ref": "#/definitions/AccessRulePropertiesSubscriptionsItem"
```

Comment:
This one I think used to be inline
<img width="1234" alt="image" src="https://github.com/user-attachments/assets/a70a543c-7def-4ac9-ba27-8fe37332da95">

maybe better if for now we can keep things structure(not introduce new definitions) as they were unless its not possible